proposal/README.md

```diff
@@ -68,25 +70,23 @@ $ flytectl --endpoint "example.flyte.net" --project "p" --domain "d" delete work
 ```
 
 ### Create is special
-Create may need more information than can be easily passed in command line and we recommend using files to create an entity. The file could be in protobuf, jsonpb (json) or jsonpb (yaml) form.
-Eventually we may want to simplify the json and yaml representations but that is not required in first pass. We may also want to create just a separate option for that.
-
-The create for Task and Workflow is essential what is encompassed in the pyflyte as the registration process. We will decouple the registration process such that pyflyte, jflyte (other native cli's or
-code methods) can dump a serialized representations of the workflows and tasks that are directly consumed by **Flytectl**. Thus Flytectl is essential in every flow for the user.
 
+The create for Task and Workflow is essential to what is encompassed in the pyflyte as the registration process. We will decouple the registration process such that the pyflyte, jflyte (other native cli's or
```

Comment:
Have you removed the previous paragraph intentionally?

Reply:
Isn't it a repetitive paragraph?

Reply:
I don't think so.

Reply:
Got it. Let me revert. Thank you.